src/apps/loan-list/list/list.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, FC, useCallback, useEffect } from "react";
+import { GetMaterialManifestationQuery } from "../../../core/dbc-gateway/generated/graphql";
+import { useText } from "../../../core/utils/text";
+import IconList from "../../../components/icon-list/icon-list";
+import IconStack from "../../../components/icon-stack/icon-stack";
+import Pagination from "../utils/pagination";
+import { ListView } from "../../../core/utils/types/list-view";
+import { LoanMetaDataType } from "../../../core/utils/types/loan-meta-data-type";
+import RenewLoansModal from "../modal/renew-loans-modal";
+import { useModalButtonHandler } from "../../../core/utils/modal";
+import modalIdsConf from "../../../core/configuration/modal-ids.json";
+import { getUrlQueryParam } from "../../../core/utils/helpers/url";
+import { isDate } from "../../../core/utils/helpers/date";
+
+export interface ListProps {
+  header: string;
+  setView: (view: string) => void;
+  selectModalMaterial: ({
+    material,
+    loanMetaData
+  }: {
+    material: GetMaterialManifestationQuery | undefined | null;
```

Comment:
I think it is better to resolve data on a higher level, so the type that could be used here could be: `Manifest` or `LoanManifest` or the like. But it is probably a bigger operation. So if you agree on it, you could also write a TODO with it.

Reply:
this will be changed